From df570a966b403a8c5a0cf22a6b29abc2f3ad81d3 Mon Sep 17 00:00:00 2001
From: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Date: Mon, 7 Nov 2022 18:04:17 +0200
Subject: [PATCH 35/49] s32g3: rdb3: Fix "ddr_utils.h" include statement

Fixed the include statement for "ddr_utils.h" to be quote-based
instead of fully-qualified, to correctly handle the case when
CUSTOM_DDR_DRV parameter is specified and the DDR Driver has to
be imported from a user-specified location.

Issue: ALB-9511
Upstream-Status: Pending 

Signed-off-by: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 plat/nxp/s32/s32g/s32g3/s32g399ardb3/s32g399ardb3_ddr_fixup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/nxp/s32/s32g/s32g3/s32g399ardb3/s32g399ardb3_ddr_fixup.c b/plat/nxp/s32/s32g/s32g3/s32g399ardb3/s32g399ardb3_ddr_fixup.c
index 297c66f1d..3c2072354 100644
--- a/plat/nxp/s32/s32g/s32g3/s32g399ardb3/s32g399ardb3_ddr_fixup.c
+++ b/plat/nxp/s32/s32g/s32g3/s32g399ardb3/s32g399ardb3_ddr_fixup.c
@@ -8,7 +8,7 @@
 #include <common/debug.h>
 #include <common/fdt_wrappers.h>
 #include <drivers/nxp/s32/adc/s32_adc.h>
-#include <drivers/nxp/s32/ddr/ddr_init.h>
+#include "ddr_init.h"
 #include <dt-bindings/clock/s32gen1-clock-freq.h>
 #include <dt-bindings/clock/s32gen1-clock.h>
 #include <lib/libc/errno.h>
-- 
2.25.1

